modules/custom/az_finder/az_finder.libraries.yml

```diff
@@ -0,0 +1,16 @@
+active-filter-count:
+  js:
+    js/active-filter-count.js:
+      - type: module
+      - minified: false
+  dependencies:
+    - core/drupal
+    - core/drupalSettings
+active-filter-reset:
+  js:
+    js/active-filter-reset.js:
+      - type: module
+      - minified: false
+  dependencies:
+    - core/drupal
+    - core/drupalSettings
```

Comment:
Very picky nitpick:  Looks like `active-filter-reset.js` isn't actually using `drupalSettings`/`settings`.
```suggestion
```

Reply:
I could swear I checked that, thanks